src/batch/src/task/task_manager.rs

```diff
@@ -164,6 +164,14 @@ impl BatchManager {
             .ok_or(TaskNotFound)?
             .get_error())
     }
+
+    /// Return the receivers for streaming RPC.
+    pub fn get_task_receiver(
```

Comment:
There is a hack: we will swap the state receiver and return as stream RPC result.